### ISSUE TITLE
SWAL2 and accessibility (VoiceOver iOS)

### ISSUE THREAD

**Issue by sacroma** [sweetalert2/sweetalert2#1206]
Hi!

When I try to use sweetalert on iPhone (iOS 11, latest) the VoiceOver (assistive tech for blind people), doesn't read the description of sweet alert modals.

Example: swal("title", "desc", "success"):
The VoiceOver reads "Ok button, 'title' modals", but no "desc".

I try the modals on bootstrap docs page and everything works, the example in sweetalert2 homepage doesn't work instead.

If you need another test or infos, ask me.
Regards

**Comment by gverni** [comment]
Hi @sacroma I did some more test on other platforms, and I can confirm that the voice over works fine on Android (tested on Pixel running Android Pie using Chrome as browser). So that may really have something to do with the way IOS "read" the screen. I'll keep investigating.

**Comment by gverni** [comment]
Hi @limonte I've done some more tests and it looks like that this issue has something to do with the animation. Using `animation: false` makes the voice over on IOS read the entire content of the modal. I'm not sure what to make out of it yet, but I think it's a step forward. 

@sacroma as a workaround you can disable the animation in the meantime. Have a try and let me know if you see the same behaviour.

**Comment by limonte** [comment]
Since the last reply the `animation` param was deprecated and removed. Disabling animation is possible with `showClass` and `hideClass` params:

```js
Swal.fire({
  title: 'SweetAlert2 modal without animation',
  showClass: {
    backdrop: 'swal2-noanimation', // disable backdrop animation
    popup: '',                     // disable popup animation
    icon: ''                       // disable icon animation
  },
  hideClass: {
    popup: '',                     // disable popup fade-out animation
  }
})
```

I tested VoiceOver on macOS, all is well. For VoiceOver on iOS please disable animations as a workaround, for now. Hopefully, Apple devs will fix issues.
